java/org/contikios/cooja/Cooja.java

```diff
@@ -2698,20 +2700,16 @@ public static void loadExternalToolsDefaultSettings() {
 
     try (var in = Cooja.class.getResourceAsStream(filename)) {
       if (in == null) {
-        throw new FileNotFoundException(filename + " not found");
+        throw new MissingResourceException(
+                "Could not find " + filename + ", jar file seems broken", "Cooja", "");
       }
       settings.load(in);
 
       currentExternalToolsSettings = settings;
       defaultExternalToolsSettings = (Properties) currentExternalToolsSettings.clone();
       logger.info("External tools default settings: " + filename);
     } catch (IOException e) {
-      logger.warn("Error when reading external tools settings from " + filename, e);
-    } finally {
-      if (currentExternalToolsSettings == null) {
-        defaultExternalToolsSettings = new Properties();
-        currentExternalToolsSettings = new Properties();
-      }
+      throw new RuntimeException(e);
```

Comment:
What do you think about throwing `MissingResourceException` instead of RuntimeException also here?